internal/cli/compile/compile.go

```diff
@@ -95,7 +120,13 @@ func NewCommand() *cobra.Command {
 	fqbnArg.AddToCommand(compileCommand)
 	profileArg.AddToCommand(compileCommand)
 	compileCommand.Flags().BoolVar(&dumpProfile, "dump-profile", false, tr("Create and print a profile configuration from the build."))
-	compileCommand.Flags().BoolVar(&showProperties, "show-properties", false, tr("Show all build properties used instead of compiling."))
+	compileCommand.Flags().StringVar(
+		&showProperties,
+		"show-properties",
+		"disabled",
+		tr(`Show build properties instead of compiling. With "pattern" properties are returned as they are defined. If "value" the placeholders are replaced with compilation context values.`),
```

Comment:
No problem changing the options. Imo `raw` sounds ambiguous though: a plain string with no placeholder syntax could be considered "raw" too.

Reply:
Some other possibilities are: `plain`, `unexpanded`, `verbatim`... I'll let you choose the best one :-)

Let's consider that 99% of the users will simply do `--show-properties` or `--show-properties=expanded`, so this word will not appear often (I would argue that it will be never used basically, IMHO it's not worth spending much time discussing it).

Reply:
went for `unexpanded`